fronts-client/src/types/Recipe.ts

```diff
@@ -19,6 +19,6 @@ export interface Recipe {
   title: string;
   canonicalArticle: string;
   difficultyLevel: string;
-  featuredImage: RecipeImage; // the latter is an old image format that appears in our test fixtures
+  featuredImage?: RecipeImage; // the latter is an old image format that appears in our test fixtures
```

Comment:
any reason for making it optional? (sorry if I missed discussion relatd to this)

Reply:
The reason I made it optional was that I came across a recipe in the "real" data where it was not set.  Now, I know that _shouldn't_ happen - but nonetheless it has.  Therefore, we need to code to deal with it because the lack of the field caused an exception.